cluster/charts/crossplane-controllers/values.yaml.tmpl

```diff
@@ -1,54 +0,0 @@
-nameOverride: crossplane
-
-replicas: 1
-
-deploymentStrategy: RollingUpdate
-
-image:
-  repository: crossplane/crossplane
-  tag: %%VERSION%%
-  pullPolicy: Always
-
-args: {}
-
-rbacManager:
-  deploy: true
-  managementPolicy: All
-
-imagePullSecrets:
-- dockerhub
-
-hostedConfig:
-  enabled: false
```

Comment:
hostedConfig.enabled needs to be cleaned up from top level values.yaml.tmpl as well: https://github.com/negz/crossplane/blob/hellm/cluster/charts/crossplane/values.yaml.tmpl#L19